src/pages/index.js

```diff
@@ -119,6 +119,9 @@ export default class extends Component {
         if (data) this.setState({ url, data })
         this.loaderStop()
       })
+      .catch(() => {
```

Comment:
I think you can write simple `.catch(this.loaderStop)`

Reply:
My bad 😅